### PR TITLE
Allow hiding the accessible view

### LINKED ISSUE #195288: [Accessibility] Allow users to configure the Accessible View transparent
Type: <b>Feature Request</b>

CC @meganrogge 

When presenting my screen or demoing my live-coding to sighted individuals, I often want to make the Accessible View invisible to my audience. This is particularly distracting for sighted audience when I use terminal and create a side-by-side visualizations. I just want to open the Accessible View for my own and make it transparent visually.

However, this does not mean that we have to make it transparent by default. I suggest adding a user configuration for them to choose between visible (default) and invisible mode.

VS Code version: Code - Insiders 1.84.0-insider (eb4f3c149f34d82b9763a08230907cf79e7fc103, 2023-10-10T07:26:46.161Z)
OS version: Windows_NT x64 10.0.22621
Modes:

### PR DESCRIPTION
**Pull request by meganrogge**
fix #195288

### CODE REVIEW

File: src/vs/workbench/contrib/codeEditor/browser/accessibility/accessibility.css [hunk 1]
```diff
@@ -52,3 +52,9 @@
 	background-repeat: no-repeat;
 	padding: 2px;
 }
+
+.accessible-view.hide {
+	position: fixed;
+	top: -2000px;
+	left:-2000px;
```
Comment (Tyriar) [inline src/vs/workbench/contrib/codeEditor/browser/accessibility/accessibility.css:59]: Does `visibility: hidden` work? Some things can go wrong when you put things off screen

**Review by Tyriar — DISMISSED** [review]

File: src/vs/workbench/contrib/codeEditor/browser/accessibility/accessibility.css (hunk 1 above)
Comment (meganrogge) [reply src/vs/workbench/contrib/codeEditor/browser/accessibility/accessibility.css:59]: nothing went wrong when I tested it, but I'll try that

Comment (meganrogge) [reply src/vs/workbench/contrib/codeEditor/browser/accessibility/accessibility.css:59]: does not work with `visibility:hidden`, focus remains in the terminal

**Review by lramos15 — APPROVED** [review]

File: src/vs/workbench/contrib/codeEditor/browser/accessibility/accessibility.css (hunk 1 above)
Comment (meganrogge) [reply src/vs/workbench/contrib/codeEditor/browser/accessibility/accessibility.css:59]: Maybe shld try opacity 0

**Pull request merged**